documentation/modules/tracing/proc-enabling-non-jaeger-tracing.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc
+
+[id='proc-enabling-non-jaeger-tracing-{context}']
+= Introducing an OpenTelemetry tracing implementation
+
+[role="_abstract"]
+By default, OpenTelemetry uses a Jaeger tracing implementation.
```

Comment:
@alesj is this really true? AFAIU (even by testing the bridge), by default it has it's own OLTP implementation in which case you need a collector running for getting data via OLTP and "translating" to Jaeger.

Reply:
True ... in OTel sense ... OTel I guess it forces you to either you're present OTEL, or discuss things fwd ....